### ISSUE TITLE
Setup Issues

### ISSUE THREAD

**Issue by Bart-1992** [Luc-S/Toon-HASS#19]
Struggling to get everything to work. ( see images )
![20191012_162658](https://user-images.githubusercontent.com/51082692/66703232-aaa96a80-ed10-11e9-80a2-304e1e988d49.jpg)
![20191012_162719](https://user-images.githubusercontent.com/51082692/66703241-be54d100-ed10-11e9-8878-ed099428b7d2.jpg)

Followed your instructions from the readme, but can't get the hassio data to the toon. The bullets are also not shown in the setup GUI( see images )? I think it has something to do with SSL or encryption, any suggestions ( i'm not an expert )? I've tried to edit the user settings trough putty : HomeassistantConfigurationScreen.qml  (to enable the SSL, but without succes).

**Comment by Luc-S** [comment]
Hi @Bart-1992 have you installed the app through the ToonStore? In that case please check to see if the latest posts have any solutions for you:
[https://www.domoticaforum.eu/viewtopic.php?f=99&t=12078&start=75](https://www.domoticaforum.eu/viewtopic.php?f=99&t=12078&start=75)

**Comment by Bart-1992** [comment]
Yes i've installed it trough the Toonstore, thanks for the suggestion! Got it working right now :)!

**Comment by Luc-S** [comment]
@Bart-1992 Ok great! :)
